### ISSUE TITLE
Enable compression

### ISSUE THREAD

**Issue by cqueern** [isitchristmas/web#129]
> Compressing resources with gzip or deflate can reduce the number of bytes sent over the network.

Hi Eric,

When you get ~5 minutes thought you might want to consider enabling compression over on isitchristmas.com to make it really pop in the browser. According to [the Google Page Speed Insights analysis for the site](https://developers.google.com/speed/pagespeed/insights/?url=isitchristmas.com), you could eliminate ~72% of the size currently used to transfer text-based resources.

More details here: 

[https://developers.google.com/speed/docs/insights/EnableCompression](https://developers.google.com/speed/docs/insights/EnableCompression)

Thanks for all your hard work.

**Comment by konklone** [comment]
Thanks, Caleb - I will do this, though probably closer to Christmas as I start gearing up to revisit hosting and infrastructure and generally start going through the backlog of fixes and fun improvements.

**Comment by cqueern** [comment]
Happy New Year Eric! 

Just a quick ping... 

Might be something you could enable before the nice weather hits the DC area again. NGINX info [here](http://nginx.org/en/docs/http/ngx_http_gzip_module.html) on compression.

![screen shot 2018-01-20 at 7 43 26 pm](https://user-images.githubusercontent.com/1588185/35189912-be528710-fe1a-11e7-92d6-69e0701e2c05.png)

**Comment by konklone** [comment]
@cqueern Thanks for prodding on this! I just went and enabled it, and verified the fix in PageSpeed.

I had gzip compression enabled, but it was default enabled for `application/x-javascript` and `text/javascript`, but not for `application/javascript` (which I'm guessing is something Express does that is reasonable but a newer style than nginx's defaults were designed around). I added `application/javascript` to the list of gzip'd mimetypes, restarted `nginx`, and that was all it took.
